package.json

```diff
@@ -70,6 +70,7 @@
     "redux-persist": "^6.0.0",
     "remarkable": "^2.0.1",
     "reselect": "^4.1.8",
+    "sass": "^1.69.6",
```

Comment:
Can you please tell me what that is?

Reply:
> Can you please tell me what that is?

Yes, when I install dependencies then I got an error about node-sass.Then someone says me that node-sass is deprecated that make me unable to install dependencies of node modules.then to fix that I install sass and then I able to install dependencies that's it.

Reply:
I just installed it to run my application because my node modules not installed makes node-sass error.

Reply:
ok